### ISSUE TITLE
Nested ViewModels

### ISSUE THREAD

**Issue by chrisaswain** [SteveSanderson/knockout-es5#6]
I have a ViewModel defined that contains two sub-ViewModels as properties like this:

```js
var vm = function () {
  var self = this;
  self.subVM1 = new SubVM1();
  self.subMV2 = new SubMV2();
  
  ko.track(self);
};
```

Both SubVM's are structured the same way as a normal VM and both call `ko.track(self)` as well.  

In my markup, I am trying to use the "with:" syntax to specify the use of one of the SubVMs like so:

```html
<div data-bind="with: SubVM1">
<h1 data-bind="text: SomePropertyOnSubVM1"></h1>
</div>
```

This binds initially just fine.  However, when `SomePropertyOnSubVM1` changes, the UI is not updated.  This worked fine when I put the property directly on the main ViewModel and not on the Sub-ViewModel.  

Any suggestions?  Is this a known issue?

**Comment by unsafecode** [comment]
+1

**Comment by nathanboktae** [comment]
This is implemented as of bcf8f67

**Comment by archangel-irk** [comment]
@nathanboktae Yes, you are right.
